project/core/templates/about.html

```diff
@@ -185,15 +186,18 @@ <h3 class="title">{% trans "Sponsors" %} </h3>
     </div>
     </main>
 </script>
+{% endblock backbone_template %}
 
-<body>
+{% block content %}
     {% include "static_nav.html" %}
     <div id="about"></div>
     {% include "static_footer.html" %}
```

Comment:
Let's move the static navigation and footer to the `base.html`.
```suggestion
    <div id="about"></div>
```